src/reanimated2/valueSetter.ts

```diff
@@ -13,12 +21,12 @@ export function valueSetter(sv: any, value: any, forceUpdate = false): void {
     typeof value === 'function' ||
     (value !== null &&
       typeof value === 'object' &&
-      (value as AnimationObject).onFrame !== undefined)
+      (value as unknown as AnimationObject).onFrame !== undefined)
```

Comment:
me not likes

Reply:
we will get to Animation Objects one day...